src/persist-client/src/batch.rs

```diff
@@ -318,6 +318,13 @@ pub(crate) const BATCH_DELETE_ENABLED: Config<bool> = Config::new(
     "Whether to actually delete blobs when batch delete is called (Materialize).",
 );
 
+/// The columnar format we write down a batch with.
```

Comment:
nit: if it's pub crate, don't need the rustdoc (the dyncfg description seems sufficient IMO)

Reply:
updated!